ocs_ci/utility/utils.py

```diff
@@ -2489,13 +2489,15 @@ def check_ceph_health_not_ok():
     sampler.wait_for_func_status(True)
 
 
-def ceph_health_check(namespace=None, tries=20, delay=30):
+def ceph_health_check(namespace=None, tries=20, delay=30, resolve_daemon_crash=True):
```

Comment:
I feel that the value resolve_daemon_crash should not be set as TRUE by default in some cases, we can actually miss a bug just by archiving it. Value should be set true for specific tests only and not default